sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -582,6 +582,15 @@ object SQLConf {
     .intConf
     .createWithDefault(2667)
 
+  val CODEGEN_MAX_CHARS_PER_FUNCTION = buildConf("spark.sql.codegen.maxCharactersPerFunction")
```

Comment:
Based on my understanding, this is not the number of characters? This is the length of source codes, right?